configs/pikamon.yml

```diff
@@ -2,5 +2,9 @@ pikamon:
   logging:
     level: debug
   bot:
-    debounce-window: 10s
-    spawn-chance: 0.05
+    debounce-window: 20s
+    spawn-chance: 0.99
+  channelCache:
```

Comment:
can this be channel-cache to be consistent with the rest of the yaml?

Reply:
done